OrcScala/src/orc/run/extensions/WorkStealingScheduler.scala

```diff
@@ -1,9 +1,7 @@
 //
-// WorkStealingScheduler.scala -- Scala trait OrcWithWorkStealingScheduler and class OrcWorkStealingExecutor
+// WorkStealingScheduler.scala -- A work-stealing scheduler for Orc
 // Project OrcScala
 //
-// Created by amp on Nov 4, 2015.
```

Comment:
Nah, I created it. I just didn't know the date and think these comments are less than useless. I'll make up some random numbers for it.